base/applications/rapps/include/rapps.h

```diff
@@ -14,8 +14,11 @@
 #include "configparser.h"
 
 extern LONG g_Busy;
+extern UINT g_PendingInstalledViewRefresh;
+extern HWND g_hLastFocus;
 
 #define WM_NOTIFY_OPERATIONCOMPLETED (WM_APP + 0)
+#define WM_NOTIFY_INSTALLERFINISHED (WM_APP + 1)
```

Comment:
nit (with respect to the previous define):
```suggestion
#define WM_NOTIFY_INSTALLERFINISHED  (WM_APP + 1)
```